File: Samples/VpSharp.VB_Sample/SayCommand.vb
Imports VpSharp.Commands
Imports VpSharp.Commands.Attributes

Public Class SayCommand
    Inherits CommandModule

    <Command("say")>
    Public Function SayAsync(context as CommandContext, <Remainder> message As String) As Task
        context.Respond(message)
        Return Task.CompletedTask
    End Function
End Class

File: Samples/VpSharp.CSharp_Sample/SayCommand.cs
using VpSharp.Commands;
using VpSharp.Commands.Attributes;

namespace VpSharp.CSharp_Sample;

public class SayCommand : CommandModule
{
    [Command("say")]
    public Task SayAsync(CommandContext context, [Remainder] string message)
    {
        context.Respond(message);
        return Task.CompletedTask;
    }
}
